### ISSUE TITLE
Catholic Codex Summit

### ISSUE THREAD

**Issue by ryanhaber** [theHsEdition/catholic-codex#21]
Any interest in having a video conference round-table discussion to brainstorm ideas and start developing a plan? Does a big plan exist yet? @theHsEdition ?

**Comment by theHsEdition** [comment]
We started writing some plans in the Wiki, so feel free to edit/add in there. Milestones need to be set for sure. I like the idea of talking it out.

**Comment by ryanhaber** [comment]
@DanielrossR @theHsEdition @gracewelch did someone create our catholiccodex gmail account yet? Was I supposed to do that?

![bitmoji](https://render.bitstrips.com/v2/cpanel/10224485-128754182_2-s1-v1.png?transparent=1&palette=1&width=246)

**Comment by DanielrossR** [comment]
I created it, but didn’t have your email or Graces, or Johns for that matter. I added Patrick and thought he was going to add y’all. Sorry about that, if y’all send me your email I’ll add y’all now. 

- Daniel Reed

**Comment by ryanhaber** [comment]
I don't have yours. Lol. Rather than post them online, @theHsEdition could you please exchange them for us?

**Comment by theHsEdition** [comment]
Yeah I invited everyone to share the folder with their personal gmail accounts about a week ago, I should have followed up in person, my bad.
